react/package.json

```diff
@@ -1,66 +1,114 @@
 {
-  "name": "ccm-stager",
```

Comment:
I'm not sure what happened here but the indentation shouldn't have changed. Did prettier overwrite this file?

Reply:
Yeah I ran prettier to format the json. Do you want me to revert? (or adjust prettier json settings)?

Reply:
Yeah, I don't think prettier should touch this. editorconfig has json and yaml at two-space indents.